lib/world_cup.rb

```diff
@@ -0,0 +1,49 @@
+class WorldCup
+
+  attr_reader :year,
+              :teams
+
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(position)
```

Comment:
This is kind of breaking SRP - it would be nice to have another method that just returns the active teams.